frontend/src/store/bucketStore.ts

```diff
@@ -77,9 +77,12 @@ export const useBucketStore = defineStore('bucket', () => {
 
         const userPermissions: UserPermissions[] = [];
         uniqueUsers.forEach((user: User) => {
+          const idp = config.value.idpList.find((idp: IdentityProvider) => idp.idp === user.idp);
+
           userPermissions.push({
             userId: user.userId,
-            elevatedRights: config.value.idpList.find((idp: IdentityProvider) => idp.idp === user.idp)?.elevatedRights,
+            idpDescription: idp?.description,
```

Comment:
We originally just ran with calling it `description` in the idpList-* file, but yes now that I think about it some more, it may be better suited to be called `name` instead.